contrib/platform/test/com/sun/jna/platform/win32/Crypt32Test.java

```diff
@@ -143,8 +149,8 @@ public void testCryptProtectUnprotectDataWithEntropy() {
                     assertTrue("CryptUnprotectData(WithEntropy)",
                             Crypt32.INSTANCE.CryptUnprotectData(pDataEncrypted, pDescription,
                                     pEntropy, null, null, 0, pDataDecrypted));
-                    assertEquals("description", pDescription.getValue().getWideString(0));
-                    assertEquals("hello world", pDataDecrypted.pbData.getString(0));
+                    assertEquals(description, pDescription.getValue().getWideString(0));
+                    assertEquals(Arrays.toString(payload), Arrays.toString(pDataDecrypted.getData()));
```

Comment:
Why are we converting arrays to strings to compare them instead of just comparing the arrays with `assertArrayEquals`?

Reply:
Good question. Changed to assertArrayEquals instead.